certbot-ci/certbot_integration_tests/certbot_tests/test_main.py

```diff
@@ -19,21 +23,63 @@ def context(request):
         integration_test_context.cleanup()
 
 
-def test_manual_http_auth(context):
```

Comment:
Did you mean to delete this test? Now its functions aren't being called.

Reply:
Not really. I put it back.